sonic_data_client/transl_data_client.go

```diff
@@ -131,7 +131,8 @@ func (c *TranslClient) StreamRun(q *queue.PriorityQueue, stop chan struct{}, w *
 	for i,sub := range subscribe.Subscription {
 		stringPaths[i] = c.path2URI[sub.Path]
 	}
-	subSupport,_ := translib.IsSubscribeSupported(stringPaths)
+	req := translib.IsSubscribeRequest{Paths:stringPaths}
```

Comment:
Does this mean, this code was broke, earlier?

Reply:
The API was changed with the previous PR in management common repo. These are corresponding changes to fix the same in telemetry PR. We were waiting on one of the telemetry PRs to be pushed.